### DISCUSSION TITLE
Can i install cuQuantum on systems other than those mentioned in the official website ?

### DISCUSSION THREAD

**Discussion by moustafa7zada** [NVIDIA/cuQuantum#144] [Q&A]
Hey! 
I don't know if this is trivial :) , but on cuQuantum's [official ](https://developer.nvidia.com/cuquantum-downloads?target_os=Linux&target_arch=x86_64)website, only specific versions of specific distros are listed as supported , I am using Fedora 37 and it reached its end of life (i can't download some stuff now ) , but i don't know if i can upgrade to fedora 39/40 cause i am afraid that cuQuantum won't work , since one of the main reason why i have linux is because of cuQuantum :)

**Comment by daniellowell** [comment]
CUDA supports fedora 39, but cuTENSOR only supports fedora 37 (in the official d[ownload page](https://developer.nvidia.com/cutensor-downloads?target_os=Linux&target_arch=x86_64&Distribution=Fedora), no mention to fedora in the [docs page support](https://docs.nvidia.com/cuda/cutensor/latest/index.html)).

Now that isn't to say it doesn't run, however, it isn't actively supported. Is it possible to switch to one of our supported distros? Thanks
